service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storageContainer/VerifyAzureStorageContainerCanBeCreatedStep.java

```diff
@@ -140,7 +140,9 @@ private StepResult validateWorkspaceStorageAccountExists(
   private StepResult validateLandingZoneSharedStorageAccountExist(
       FlightContext context, AzureCloudContext azureCloudContext, StorageManager storageManager) {
```

Comment:
After the change do we need to pass the azureCloudContext as a parameter here?

Reply:
Yes. It is used in this method

Reply:
I will remove it.